activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -213,7 +213,7 @@ def initialize(connection, logger, connection_parameters, config)
         @statements = StatementPool.new @connection,
                                         self.class.type_cast_config_to_integer(config.fetch(:statement_limit) { 1000 })
 
-        if postgresql_version < 80200
+        if postgresql_version < 80300
```

Comment:
Yes, already Rails 5 does not work on PostgreSQL 9.0 and below. See the discussion: #23234